bin/oneoff/populate_metrics_from_tsv

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env ruby
+
+# This script reads a TSV of (INTEGER) data and writes the data to the metrics
+# table.
+
+require_relative '../../dashboard/config/environment'
+require 'csv'
+
+# Prompts the user for the TSV file and reads it. The file is expected to have
+# two columns, a column of dates (metric_on) and a column of values, as well as
+# a header row.
+# @return data [Array]
+def get_data
+  print 'TSV FILE: '
+  tsv_file = gets.chomp
+
+  data = []
+  CSV.foreach(tsv_file, headers: true, col_sep: "\t") do |row|
+    data << row
+    puts row
+  end
+  data
+end
+
+# Prompts the user for metadata about the data.
+# @return [Hash] The metadata, as a hash.
+def get_metadata
+  print 'COMPUTED ON: '
+  computed_on = gets.chomp
+
+  print 'COMPUTED BY: '
+  computed_by = gets.chomp
```

Comment:
The actual insert statement uses strings, so casting them to dates (to get casted back to strings) is unnecessary.